index.html

```diff
@@ -6,23 +6,53 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Albert's Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link
+			href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap"
+			rel="stylesheet"
+		/>
 </head>
 <body>
   <header class="header-container">
     <a href="#" class="logo">
         My Logo
     </a>
     <nav class="logo-button">
-        <a href="#"><img src="images/logo-button.png" alt="menu" width="18px" height="12px"></a>
+        <a href="#" class="humburger"><img src="images/logo-button.png" alt="menu" width="18px" height="12px"></a>
```

Comment:
- [OPTIONAL] This is not part of the requirement for this milestone, but I'll suggest you fix this.
- Currently, your portfolio has a `max-width` property for most of the sections, looking at the screenshot below, you can see that on screens above `1440 px` , there is a white space by the right. this is not good for the UI and makes your project look a bit unprofessional. I'll suggest you go through your project and remove all the `max-width` you added in the media query. :+1: :+1: 
- - Screenshot for reference:
![Screenshot from 2023-06-21 19-07-10](https://github.com/albertkantwi/Portfolio_Project/assets/107632780/68cf9f72-0097-4826-b71f-3984a67d62e3)